code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -40,6 +40,16 @@
 	)
 	biohazard = 45
 
+	minimal_lightup_areas = list(
+		/area/crew_quarters/heads/cmo,
+		/area/storage/eva,
+		/area/medical/morgue,
+		/area/medical/surgery,
+		/area/medical/chemistry,
+		/area/medical/apothecary,
+		/area/medical/genetics/cloning
```

Comment:
Note: having `genetics/cloning` instead of just `/cloning` here is intentional: I've only _very rarely_ seen CMOs do genetics, at least compared to normal doctors and such

Reply:
it should still be included, as we removed RD lording over genetics and made it wholly CMO's territory.

Reply:
Alright, included all of genetics now.